### PR TITLE
Update dashboarding section for new features

### PR DESCRIPTION
**Pull request by leifmadsen**
Update the dashboarding module to sync with the new functionality merged in
https://github.com/infrawatch/service-telemetry-operator/pull/115

### CODE REVIEW

**Comment by leifmadsen** [comment]
[Update graphing section in documentation for new `graphingEnabled` functionality in STO](https://trello.com/c/tSNifSD7/1115-update-graphing-section-in-documentation-for-new-graphingenabled-functionality-in-sto)

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 1]
```diff
@@ -23,7 +23,13 @@
 [id="setting-up-grafana-to-host-the-dashboard_{context}"]
 = Setting up Grafana to host the dashboard
 
-Grafana is not included in the default {Project} ({ProjectShort}) deployment so you must deploy the Grafana Operator from OperatorHub.io.
+Grafana is not included in the default {Project} ({ProjectShort}) deployment so you must deploy the Grafana Operator from OperatorHub.io. Using the Service Telemetry Operator to deploy Grafana will result in a Grafana instance
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:26]:
```suggestion
Grafana is not included in the default {Project} ({ProjectShort}) deployment so you must deploy the Grafana Operator from OperatorHub.io. Using the Service Telemetry Operator to deploy Grafana results in a Grafana instance and the  configuration of the default data sources for the local {ProjectShort} deployment.
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 2]
```diff
@@ -23,7 +23,13 @@
...
+and the default datasources configured for the local {ProjectShort} deployment.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:27]:
```suggestion
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 3]
```diff
@@ -23,7 +23,13 @@
...
+
+[discrete]
+.Prerequisites
+
+Enable OperatorHub.io catalog source for the Grafana Operator. See xref:enabling-the-operatorhubio-community-catalog-source[].
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:32]:
```suggestion
Enable OperatorHub.io catalog source for the Grafana Operator. For more information, see xref:enabling-the-operatorhubio-community-catalog-source[].
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 4]
```diff
@@ -23,7 +23,13 @@
...
-Grafana is not included in the default {Project} ({ProjectShort}) deployment so you must deploy the Grafana Operator from OperatorHub.io.
+Grafana is not included in the default {Project} ({ProjectShort}) deployment so you must deploy the Grafana Operator from OperatorHub.io. Using the Service Telemetry Operator to deploy Grafana will result in a Grafana instance
+and the default datasources configured for the local {ProjectShort} deployment.
+
+[discrete]
+.Prerequisites
+
+Enable OperatorHub.io catalog source for the Grafana Operator. For more information, see xref:enabling-the-operatorhubio-community-catalog-source[].
 
 [discrete]
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:34]:
```suggestion
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 5]
```diff
@@ -36,89 +41,140 @@ Grafana is not included in the default {Project} ({ProjectShort}) deployment so
...
++
+[source,bash,options="nowrap"]
+----
 NAME                                DISPLAY                                         VERSION   REPLACES                            PHASE
 grafana-operator.v3.2.0             Grafana Operator                                3.2.0                                         Succeeded
 ...
 ----
 
-. Launch a Grafana instance:
+. Launch a Grafana instance by creating or modifying the `ServiceTelemetry` object. Set `graphingEnabled` to `true`.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:77]:
```suggestion
. To launch a Grafana instance, create or modify the `ServiceTelemetry` object. Set `graphingEnabled` to `true`.
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 6]
```diff
@@ -36,89 +41,140 @@ Grafana is not included in the default {Project} ({ProjectShort}) deployment so
...
-
+----
++
+----
 NAME                                  READY   STATUS    RESTARTS   AGE
 grafana-deployment-7fc7848b56-sbkhv   1/1     Running   0          1m
 ----
 
-. Create the datasource and dashboard resources:
+[discrete]
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:106]:
```suggestion
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 7]
```diff
@@ -36,89 +41,139 @@ Grafana is not included in the default {Project} ({ProjectShort}) deployment so
...
+
+* For more information about enabling the OperatorHub.io catalog source, see xref:enabling-the-operatorhubio-community-catalog-source[].
+
+
+[id="importing-dashboards_{context}"]
+= Importing dashboards
+
+Dashboards can be imported and managed by the Grafana Operator by creating `GrafanaDashboard` objects. Example dashboards are provided at https://github.com/infrawatch/dashboards.
+
+[discrete]
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:116]:
```suggestion
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 8]
```diff
@@ -36,89 +41,138 @@ Grafana is not included in the default {Project} ({ProjectShort}) deployment so
...
-
-* For more information about enabling the OperatorHub.io catalog source, see xref:enabling-the-operatorhubio-community-catalog-source[].
-
-
 
 [id="viewing-and-editing-queries"]
-== Viewing and editing queries
+= Viewing and editing queries
 
 [discrete]
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:177]:
```suggestion
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 9]
```diff
@@ -36,91 +41,139 @@ Grafana is not included in the default {Project} ({ProjectShort}) deployment so
...
++
+----
 NAME                                  READY   STATUS    RESTARTS   AGE
 grafana-deployment-7fc7848b56-sbkhv   1/1     Running   0          1m
 ----
 
-. Create the datasource and dashboard resources:
+.Additional resources
+
+* For more information about enabling the OperatorHub.io catalog source, see xref:enabling-the-operatorhubio-community-catalog-source[].
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:108]:
```suggestion

```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 10]
```diff
@@ -36,91 +41,139 @@ Grafana is not included in the default {Project} ({ProjectShort}) deployment so
...
-
+----
++
+----
 NAME                                  READY   STATUS    RESTARTS   AGE
 grafana-deployment-7fc7848b56-sbkhv   1/1     Running   0          1m
 ----
 
-. Create the datasource and dashboard resources:
+.Additional resources
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:106]:
```suggestion

```

Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:106]: We can remove this section as it's already in the Prereqs above

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 11]
```diff
@@ -36,91 +41,139 @@ Grafana is not included in the default {Project} ({ProjectShort}) deployment so
...
+
+* For more information about enabling the OperatorHub.io catalog source, see xref:enabling-the-operatorhubio-community-catalog-source[].
+
+
+[id="importing-dashboards_{context}"]
+= Importing dashboards
+
+Dashboards can be imported and managed by the Grafana Operator by creating `GrafanaDashboard` objects. Example dashboards are provided at https://github.com/infrawatch/dashboards.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:114]:
```suggestion
The Grafana Operator can import and manage dashboards by creating `GrafanaDashboard` objects. You can view example dashboards at https://github.com/infrawatch/dashboards.
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 12]
```diff
@@ -36,91 +41,139 @@ Grafana is not included in the default {Project} ({ProjectShort}) deployment so
...
-NAME                                  AGE
-service-telemetry-grafanadatasource   1m
+----
++
+----
+NAME                        AGE
+stf-default-ds-prometheus   20h
+----
+
+. Expose the grafana service as a route
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:151]:
```suggestion
. Expose the grafana service as a route:
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 13]
```diff
@@ -36,91 +41,139 @@ Grafana is not included in the default {Project} ({ProjectShort}) deployment so
...
+
+
+[id="importing-dashboards_{context}"]
+= Importing dashboards
+
+Dashboards can be imported and managed by the Grafana Operator by creating `GrafanaDashboard` objects. Example dashboards are provided at https://github.com/infrawatch/dashboards.
+
+.Procedure
+
+. Import a dashboard.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:118]:
```suggestion
. Import a dashboard:
```

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc [hunk 14]
```diff
@@ -36,91 +41,139 @@ Grafana is not included in the default {Project} ({ProjectShort}) deployment so
...
+----
+
+. Expose the grafana service as a route:
++
+[source,bash,options="nowrap"]
+----
+oc create route edge dashboards --service=grafana-service --insecure-policy="Redirect" --port=3000
 ----
 
 . Navigate to https://<grafana-route-address> in a web browser. Use the `oc get routes` command to retrieve the Grafana route address:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:158]:
```suggestion
. Navigate to https://<grafana-route-address> in a web browser. 

. Use the `oc get routes` command to retrieve the Grafana route address:
```

Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:158]: @leifmadsen Apologies if this came up previously. How can one navigate to an address that they do not have yet? 
Should this be separated out, something like: 

. Use the `oc get routes` command to retrieve the Grafana route address:
**comand and output blah**
The HOST/PORT value is the Grafana route address. 
. Navigate to https://<grafana-route-address> in a web browser. Replace <grafana-route-address> with the HOST/PORT value you retrieved in the previous step.

**Review by JoanneOFlynn2018 — CHANGES_REQUESTED** [review]
I committed minor edits and left suggested edits/comments on anything else I thought you would like to see!

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc (hunk 14 above)
Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc:155]: Ah ha, yes, that sounds right. Use `oc get routes` to obtain the address first, then navigate to https://<value from HOST/PORT>.

**Comment by leifmadsen** [comment]
All changes from @JoanneOFlynn2018 look good to me, and I've merged those into the PR. Looks like just the one outstanding change to do.

**Review by JoanneOFlynn2018 — APPROVED** [review]

**Pull request merged**